### ISSUE TITLE
Export-M365DSCConfiguration directly to the Azure blob container

### ISSUE THREAD

**Issue by JyrkiHuikuri** [microsoft/Microsoft365DSC#4794]
Is it possible to save the configuration with a command
Export-M365DSCConfiguration directly to the Azure blob container when the command is run from the Azure runbook.

**Comment by FabienTschanz** [comment]
@JyrkiHuikuri No, that's not possible. What you can do is export the configuration, zip it and then upload the configuration to the storage account (or don't zip it and upload all files, but for that a share is probably better than a blob). 

Check out [Upload blobs to the container](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-powershell#upload-blobs-to-the-container) for further information on how to upload files to a blob container.
